lib/sass.dart

```diff
@@ -33,3 +33,23 @@ String render(String path,
       evaluate(sassTree, color: color, packageResolver: packageResolver);
   return toCss(cssTree);
 }
+
+/// Loads [source] as Sass, converts it to CSS, and returns the result.
+///
+/// If [color] is `true`, this will use terminal colors in warnings.
+///
+/// If [packageResolver] is provided, it's used to resolve `package:` imports.
+/// Otherwise, they aren't supported. It takes a [SyncPackageResolver][] from
+/// the `package_resolver` package.
+///
+/// [SyncPackageResolver]: https://www.dartdocs.org/documentation/package_resolver/latest/package_resolver/SyncPackageResolver-class.html
+///
+/// Finally throws a [SassException] if conversion fails.
+String renderSource(String source,
+    {bool color: false, SyncPackageResolver packageResolver}) {
+  var url = new UriData.fromString(source);
```

Comment:
I suspect using a `data:` URL here will produce some pretty gnarly errors. We should allow the user to pass in their own URL if they want, but if they don't I think we should just forward the `null` value to `parseScss()`. If it chokes, that's a bug.